ppa/archive/eebo_tcp.py

```diff
@@ -28,26 +28,100 @@ class Page(xmlmap.XmlObject):
     # use following axis to find all text nodes following this page beginning
     text_contents = xmlmap.StringListField("following::text()")
 
+    # notes on or after this page, for managing footnote text
```

Comment:
My xmlobject class here is named `Page` but that's somewhat inaccurate, since in tei pages aren't containers but only delimited by `<PB>` tags. This xpath will find _all_ notes after the current page / pb. In a volume with a lot of notes that might be problematic, I'm not sure. I was wondering about looking to see if it's possible to filter the xpath to only find notes after this pb and before the next one, but I didn't go looking.